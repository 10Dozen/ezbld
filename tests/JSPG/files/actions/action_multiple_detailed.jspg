@ Action 1 | dialog_right pic.jpg | MyTag
*scene: CurrentScene
*goto: NextScene

@ Action 2
*scene: CurrentScene
*goto: NextScene

@ Action 3 | scene
*scene: CurrentScene
*goto: NextScene

@ Action 4 || MyTag2
*scene: CurrentScene
*goto: NextScene

@ Action 5 |
*scene: CurrentScene
*goto: NextScene

@ Action 6 | scene_left |
*scene: CurrentScene
*goto: NextScene

@ Action 7 ||
*scene: CurrentScene
*goto: NextScene
