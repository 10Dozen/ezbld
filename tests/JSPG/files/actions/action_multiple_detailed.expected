[0]
cls = JSPGAction
name = Action 1 | TEXT
scene = CurrentScene | TEXT
type = dialog_right | TEXT
portrait = pic.jpg | TEXT
tag = MyTag | TEXT
goto = NextScene | TEXT

[1]
cls = JSPGAction
name = Action 2 | TEXT
scene = CurrentScene | TEXT
goto = NextScene | TEXT

[2]
cls = JSPGAction
name = Action 3 | TEXT
scene = CurrentScene | TEXT
type = scene | TEXT
goto = NextScene | TEXT

[3]
cls = JSPGAction
name = Action 4 | TEXT
scene = CurrentScene | TEXT
tag = MyTag2 | TEXT
goto = NextScene | TEXT

[4]
cls = JSPGAction
name = Action 5 | TEXT
scene = CurrentScene | TEXT
goto = NextScene | TEXT

[5]
cls = JSPGAction
name = Action 6 | TEXT
scene = CurrentScene | TEXT
type = scene_left | TEXT
goto = NextScene | TEXT

[6]
cls = JSPGAction
name = Action 7 | TEXT
scene = CurrentScene | TEXT
goto = NextScene | TEXT